pkgs/applications/networking/resilio-sync/default.nix

```diff
@@ -10,14 +10,14 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "resilio-sync";
-  version = "2.7.2";
+  version = "2.7.3";
 
   src = fetchurl {
     url = "https://download-cdn.resilio.com/${version}/linux-${arch}/resilio-sync_${arch}.tar.gz";
     sha256 = {
-      x86_64-linux = "0gar5lzv1v4yqmypwqsjnfb64vffzn8mw9vnjr733fgf1pmr57hf";
-      i686-linux   = "1bws7r86h1vysjkhyvp2zk8yvxazmlczvhjlcayldskwq48iyv6w";
-      aarch64-linux = "0j8wk5cf8bcaaqxi8gnqf1mpv8nyfjyr4ibls7jnn2biqq767af2";
+      x86_64-linux = "sha256-DYQs9KofHkvtlsRQHRLwQHoHwSZkr40Ih0RVAw2xv3M=";
+      i686-linux = "sha256-PFKVBs0KthG4tuvooHkAciPhNQP0K8oi2LyoRUs5V7I=";
+      aarch64-linux = "sha256-o2DlYOBTkFhQMEDJySlVSNlVqLNbBzacyv2oTwxrXto=";
     }.${stdenv.hostPlatform.system};
```

Comment:
This is missing an or throw unsupported platform.

Reply:
Wouldn't that be caught by this block:
https://github.com/NixOS/nixpkgs/blob/732d35fbf005b8f9b17c502b7c7836aa496dab01/pkgs/applications/networking/resilio-sync/default.nix#L4-L8

Reply:
I don't know tbh.

Reply:
I've cleaned up the code block.